virtool/startup.py

```diff
@@ -359,3 +362,15 @@ async def init_version(app: typing.Union[dict, aiohttp.web.Application]):
     logger.info(f"Mode: {app['mode']}")
 
     app["version"] = version
+
+
+async def init_tasks(app: aiohttp.web.Application):
+    db = app["db"]
```

Comment:
Don't run the task if the `no-check-db` option is set. This will prevent the ID clash in the tests as the test server has the option set by default.

See here for example of how this is used to skip DB checks in `virtool.startup`: https://github.com/virtool/virtool/blob/031c1e75ca29ddf5f1e34039ab035aab608c65dd/virtool/startup.py#L62